xtask/src/check.rs

```diff
@@ -14,7 +14,7 @@ const EXTRA_LINTS: &[&str] = &[
     "clippy::fn_to_numeric_cast_any",
     "clippy::ptr_cast_constness",
     // == Correctness == //
-    "unused_tuple_struct_fields",
+    "dead_code",
```

Comment:
This is a Rust 1.77.0 change. It's not clear which version is expected, perhaps that should be documented somewhere (or maybe it is, but I missed it)?

Reply:
Ah, I see we have a `rust-toolchain.toml`, never mind!